client-app-example/ifc-cli.config.js

```diff
@@ -15,11 +15,13 @@ module.exports = function(frameRouter) {
     {
       locale: 'en-US',
       hostRootUrl: window.location.origin + '/#/',
+      /*
```

Comment:
Did you want to keep this commented out?

Reply:
I dd not! Good catch!